ldelf/ta_elf_rel.c

```diff
@@ -429,6 +429,37 @@ static void e64_process_dyn_rela(const Elf64_Sym *sym_tab, size_t num_syms,
 	*where = val;
 }
 
+static void e64_get_sym_name(const Elf64_Sym *sym_tab, size_t num_syms,
+			     const char *str_tab, size_t str_tab_size,
+			     Elf64_Rela *rela, const char **name)
+{
+	size_t sym_idx = 0;
```

Comment:
This looks like copy&paste from `e64_process_dyn_rela()`. Perhaps we can refactor instead?

Reply:
Agreed